src/components/router/index.js

```diff
@@ -8,8 +8,8 @@ const Routing = () => {
     return <Router>
         <Header />
         <Switch>
-            <Route exact path='/text-encryptor/' component={Generate} />
-            <Route exact path='/text-encryptor/messages/:id' component={Message} />
+            <Route exact path='/' component={Generate} />
```

Comment:
let's replace '/' by `HOMEPAGE_PATH` so that if we need to make a change in the future, we can just update `HOMEPAGE_PATH`